src/stt/src/SteinerTreeBuilder.cpp

```diff
@@ -67,8 +70,22 @@ Tree SteinerTreeBuilder::makeSteinerTree(odb::dbNet* net,
                                          std::vector<int>& y,
                                          int drvr_index)
 {
-  float net_alpha = net_alpha_map_.find(net) != net_alpha_map_.end() ?
-                    net_alpha_map_[net] : alpha_;
+  float net_alpha = alpha_;
+
+  if (min_fanout_alpha_.first >= 0) {
+    int num_pins = net->getTermCount();
+    net_alpha = num_pins >= min_fanout_alpha_.first ?
+                net_alpha = min_fanout_alpha_.second : net_alpha;
+  }
```

Comment:
This would be simpler if the unset value was MAXINT rather than -1.  Then you could just do
`
  if (net->getTermCount() >= min_fanout_alpha_.first) {
    net_alpha = min_fanout_alpha_.second ;
  }
`